Test/Utils/ContrailAPI_New/Models/VirtualNetwork.ps1

```diff
@@ -1,16 +1,17 @@
 # Those are just informative to show dependencies
 #include "Subnet.ps1"
 #include "NetworkPolicy.ps1"
+#include "Tag.ps1"
 
 class VirtualNetwork : BaseResourceModel {
     [Subnet] $Subnet
     [FqName] $IpamFqName = [FqName]::new(@("default-domain", "default-project", "default-network-ipam"))
     [FqName[]] $NetworkPolicysFqNames = @()
-
+    [FqName] $TagFqName
     [String] $ResourceName = 'virtual-network'
     [String] $ParentType = 'project'
 
-    VirtualNetwork([String] $Name, [String] $ProjectName, [Subnet] $Subnet) {
+    hidden [void] init([String] $Name, [String] $ProjectName, [Subnet] $Subnet) {
```

Comment:
As talked IRL, Init should take all possible parameters and constructors just set default values.